### PR TITLE
V7 update

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
- update versions
- update services url
- add help > migration

### CODE REVIEW

**Comment by PierreJeanjacquot** [comment]
preview https://documentations.iex.ec/staging/v/v7-update

File: help/migration.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  * [Application migration](migration#application-migration)
+  * [Dataset migration](migration#dataset-migration)
+  * [Requester migration](migration#requester-migration)
+* [JS library](migration#js-library)
+
+## CLI users
+
+### Application migration
+
+Application developers needs to publish their application sell order to the v7 marketplace in order to makes it availlable for requesters.
```
Comment (james-toussaint) [inline help/migration.md]:
```suggestion
Application developers needs to publish their application sell order to the v7 marketplace in order to make it availlable for requesters.
```

File: help/migration.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Publish your application to the v7 marketplace:
+
+```bash
+iexec app publish <app-address> [options]
+```
+
+### Dataset migration
+
+Dataset developers needs to push their dataset secret to the v7 SMS and publish their dataset sell order to the v7 marketplace in order to makes it availlable for requesters.
```
Comment (james-toussaint) [inline help/migration.md]:
```suggestion
Dataset developers needs to push their dataset secret to the v7 SMS and publish their dataset sell order to the v7 marketplace in order to make it availlable for requesters.
```

File: help/migration.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Publish your dataset to the v7 marketplace:
+
+```bash
+iexec dataset publish <dataset-address> [options]
+```
+
+### Requester migration
+
+Requesters needs to login to the Result proxy v7 to store theire results and must push their encryption public key to the v7 SMS to use results encryption feature.
```
Comment (james-toussaint) [inline help/migration.md]:
```suggestion
Requesters need to login to the Result proxy v7 to store their results and must push their encryption public key to the v7 SMS to use result encryption feature.
```

**Review by james-toussaint — APPROVED** [review]
:+1:

**Pull request merged**